Cargo.toml

```diff
@@ -19,18 +19,22 @@ readme = "README.md"
 all-features = true
 
 [workspace]
-members = [ "derive" ]
+members = ["derive"]
 
 [dependencies]
 byteorder = "1"
 py_literal = "0.4"
-zip = { version = "0.6", optional = true }  # NOTICE: also in dev-dependencies
+zip = { version = "0.6", optional = true } # NOTICE: also in dev-dependencies
 num-bigint = "0.4"
 
+# required for f16 support
+half = { version = "2.3.1", optional = true }
```

Comment:
Wait, lmao actually I'm confused, the public dep links in lib.rs are already guaranteed to be in sync because I'm using ``[`those::cool_ass::smart_links`]``... 🤔

...that comment might be old. 😅